cli/src/main/java/Main.java

```diff
@@ -3,234 +3,280 @@
-      System.out.printf("%-20s %-20s %-20s %-20s %-20s %n",
-                        stats.getTotalRolls(),
-                        stats.getAvgRollsPerGame(),
-                        stats.getCumulativeScore(),
-                        stats.getAvgScore(),
-                        stats.getAvgNumDiceUsed());
+        //Header
+        System.out.printf("%-20s %-20s %-20s %-20s %-20s%n",
+                          "Total Rolls",
+                          "Avg Rolls Per Game",
+                          "Cumulative Score",
+                          "Avg Score",
+                          "Avg # Of Dice Used");
+        //Data
+        System.out.printf("%-20s %-20s %-20s %-20s %-20s%n",
+                          stats.getTotalRolls(),
+                          stats.getAvgRollsPerGame(),
+                          stats.getCumulativeScore(),
+                          stats.getAvgScore(),
+                          stats.getAvgNumDiceUsed());
     }
-    
+
     public static int changeDice() {
-         Scanner scan = new Scanner(System.in);
-         int numDice = 0;
-         
-         do {
+        Scanner scan = new Scanner(System.in);
+        int numDice = 0;
+
+        do {
             System.out.print("\nEnter number of dice (1-3): ");
-            
+
             numDice = getIntInput();
-            
-         } while (!validIntInput(numDice, 1, 3));
-         
-         return numDice;
+
+        } while (!validIntInput(numDice, 1, 3));
+
+        return numDice;
     }
-    
+
     public static int getIntInput() {
-      Scanner scan = new Scanner(System.in);
-      int input = 0;
-      
-      while (!scan.hasNextInt()) {
-         System.out.println("\nError: Input must be an integer, please try again!");
-         scan.next(); //Clears the new line character
-      }
-      
-      input = scan.nextInt();
-      
-      return input;
+        Scanner scan = new Scanner(System.in);
+        int input = 0;
+
+        while (!scan.hasNextInt()) {
+            System.out.println("\nError: Input must be an integer, try again!");
+            scan.next(); //Clears the new line character
+        }
+
+        input = scan.nextInt();
+
+        return input;
     }
-    
+
     public static boolean validIntInput(int input, int min, int max) {
-      if (input < min || input > max) {
-         System.out.println("\nError: Invalid option, try again!");
-         return false;
-      }
-      return true;
+        if (input < min || input > max) {
+            System.out.println("\nError: Invalid option, try again!");
+            return false;
+        }
+        return true;
+    }
+
+    public static String getInitials() {
+        Scanner scan = new Scanner(System.in);
+        String input;
+        boolean invalidInitials;
+
+        do {
+            System.out.println("\nEnter your initials (3 characters): ");
+            input = scan.nextLine();
+
+            if (input.length() != 3 || !input.matches("[a-zA-Z]+")) {
```

Comment:
`DiceGame.create()` will throw an exception if the username is invalid. Would it be too much trouble to refactor things so that you can `create()` in a loop as long as it keeps throwing an exception? That way we don't have to duplicate checking the length here since we're already doing it on the backend.